core/services/relay/evm/evmtesting/chain_writer_historical_wrapper.go

```diff
@@ -0,0 +1,39 @@
+package evmtesting
+
+import (
+	"context"
+	"math/big"
+
+	commontypes "github.com/smartcontractkit/chainlink-common/pkg/types"
+	interfacetesttypes "github.com/smartcontractkit/chainlink-common/pkg/types/interfacetests"
+	primitives "github.com/smartcontractkit/chainlink-common/pkg/types/query/primitives"
+)
+
+// This wrapper is required to enable the ChainReader to access historical data
+// Since the geth simulated backend doesn't support historical data, we use this
+// thin wrapper.
```

Comment:
What test(s) use historical data? 

I feel like this shouldn't be reusable, if we need this ability to assure compatibility, it's ok to mock it for the simulated backend but not for a real chain.  Therefore, I would put it in evm/ and end with _test so it's only usable by that test suite.

Reply:
> What test(s) use historical data?

The ones which test the `ChainReader`'s confidence level based reading I believe. But you're right, these tests _are_ only use by the Simulated Backend and thus can be moved to testing files in `/evm`

Reply:
Moved the two wrappers two `/evm` with `_test` suffixes